### PR TITLE
Serenity test for Blockstack Explorer (In Progress)

### PR DESCRIPTION
**Pull request by timstackblock**
Here is the first pass at automation for the Blockstack explorer.

The current suite of test will fail when we attempt to search by block or do anything that is documented in the current Blockstack Explorers Smoke test. Once this code is fixed the test will pass.

### CODE REVIEW

File: serenity-develop/features/blockstack/step_definitions/explorer.steps.ts [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+
+module.exports = function myStepDefinitions () {
+    const homePage: HomePage = new HomePage();
+    this.setDefaultTimeout(60 * 1000);
+
+    this.Given(/^user is on home page$/, async ()=> {
+        homePage.openHomePage();
+      });
+      this.Then('user enter the name in search bar', async ()=> {
+        await homePage.enterNameInSearchBar("aaronbailey.id");
```
Comment (kantai) [inline serenity-develop/features/blockstack/step_definitions/explorer.steps.ts]: `aaronbailey.id` is an id that may expire in the future (if the owner doesn't renew the name) -- using a subdomain like name (e.g., `aaron.id.blockstack`) or a name like `jude.blockstack` which doesn't expire will prevent this from raising a false error in the future.

Comment (timstackblock) [reply serenity-develop/features/blockstack/step_definitions/explorer.steps.ts]: Updated

**Comment by timstackblock** [comment]
Serenity test are running on this branch they are still failing as the new code hasn't been pushed yet.

**Comment by hstove** [comment]
I'm closing this, because the code is in #114

**Pull request closed without merging**